kcbq-confluent/src/main/java/com/wepay/kafka/connect/bigquery/schemaregistry/schemaretriever/SchemaRegistrySchemaRetriever.java

```diff
@@ -79,9 +108,33 @@ public Schema retrieveSchema(TableId table, String topic, KafkaSchemaRecordType
   }
 
   @Override
-  public void setLastSeenSchema(TableId table, String topic, Schema schema) { }
+  public void setLastSeenSchema(TableId table, TopicAndRecordName topicAndRecordName, Schema schema) {
+  }
+
+  private Map<String, TopicAndRecordName> getSubjects(List<String> topics, Map<Pattern, String> recordAliases) {
```

Comment:
Can you explain the idea here about getting subjects from `topics` & `recordNamePatterns`?

Reply:
Before the explanation begins, I'd like to tell that I'm really glad that we are going to create all tables on the fly. Getting rid of this schemas discovery is a relief.

Below is the copy paste from an original PR, answering the same question:
> We are using Kafka topics to carry on some events, like user logged in, user paid, admin removed an account, etc. Each type of an event has its own record type and we rely on TopicRecordNameStrategy (io.confluent.kafka.serializers.subject.TopicRecordNameStrategy).
This might sound like a hack while it's one of the fully supported use cases (see https://www.confluent.io/blog/put-several-event-types-kafka-topic/).
> 
> Number of unique event types (record types) might be humongous and increase in time (usually new feature introduces a couple of events). Maintaining a list of all supported record types for a topic in the connector's configuration would not only be cumbersome but also would require restarts and reconfiguration whenever you introduce new record type (which becomes more problematic when using some hosted kafka solutions).
> 
> That's why I decided to rely on regular expressions which let me declare one or a couple of patterns (instead of dozens or hundreds) to rule them all. Also since creating missing tables on the fly is supported, there is no need for restarts.
> 
> Unfortunately we cannot query schema reqistry using those patterns and we do not know exact record names in advance.
The only solution is to fetch them all and filter, using regular expressions defined in the config.